File: Solutions/1550. Three Consecutive Odds/1550. Three Consecutive Odds.go
func threeConsecutiveOdds(arr []int) bool {
    cnt:=0
    for i:=range(arr){
        if arr[i]%2==0{
            cnt=0
        }else{
            cnt+=1
            if cnt==3{
                return true
            }
        }
    }
    return false
}

File: Solutions/1550. Three Consecutive Odds/1550. Three Consecutive Odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        for(int i=0,cnt=0;i<arr.size();++i){
            if(arr[i]%2==0)
                cnt=0;
            else if(++cnt==3)return true;
        }
        return false;
    }
};